core/Plugin/Manager.php

```diff
@@ -1080,6 +1088,13 @@ private function installPluginIfNecessary(Plugin $plugin)
             $updater = new Updater();
             $updater->markComponentSuccessfullyUpdated($plugin->getPluginName(), $plugin->getVersion());
             $saveConfig = true;
+
+            /**
+             * Event triggered after a new plugin has been installed.
+             *
+             * @param string $pluginName The plugin that has been installed.
+             */
+            Piwik::postEvent('PluginManager.pluginInstalled', array($plugin));
```

Comment:
For consistency with pluginunistalled, does it make sense to use `$pluginName` here as well? if $plugin is needed, could use it as a second param.

The docs need to be adjusted as it is currently `Plugin $plugin` not `string $pluginName`.

Maybe we could also add an example to all the new events with an example use case? We have this for many other events and it will be possible on the developer site.

Should we mention that this event may be triggered several times when the config file is not writable?